js/src/lib/inferenceSnippets/serveCurl.ts

```diff
@@ -5,32 +5,39 @@ export const bodyBasic = (model: ModelData): string =>
 	`-d '{"inputs": ${getModelInputSnippet(model, true)}}'`;
 
 export const bodyZeroShotClassification = (model: ModelData): string =>
-	`-d '{"inputs": ${getModelInputSnippet(model, true)}, "parameters": {"candidate_labels": ["refund", "legal", "faq"]}}'`;
+	`-d '{"inputs": ${getModelInputSnippet(
+		model,
+		true
+	)}, "parameters": {"candidate_labels": ["refund", "legal", "faq"]}}'`;
 
-export const curlSnippetBodies:
-	Partial<Record<keyof typeof PipelineType, (model: ModelData) => string>> =
-{
+export const curlSnippetBodies: Partial<
+	Record<keyof typeof PipelineType, (model: ModelData) => string>
+> = {
 	// Same order as in js/src/lib/interfaces/Types.ts
-	"text-classification":      bodyBasic,
-	"token-classification":     bodyBasic,
+	"text-classification": bodyBasic,
+	"token-classification": bodyBasic,
 	"table-question-answering": bodyBasic,
-	"question-answering":       bodyBasic,
```

Comment:
we want these extra spaces, no?

what eslint or prettier settings would allow me to keep those spaces? I've tried using moon-landing eslint setting with no success